<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>TopicQuestions</class>
    <widget class="QWidget" name="TopicQuestions">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>TopicQuestions</string>
        </property>
        <widget class="QListWidget" name="listWidget">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>20</y>
                    <width>361</width>
                    <height>231</height>
                </rect>
            </property>
        </widget>
        <widget class="QPushButton" name="deleteButton">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>260</y>
                    <width>171</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="text">
                <string>Delete</string>
            </property>
        </widget>
        <widget class="QPushButton" name="goBackButton">
            <property name="geometry">
                <rect>
                    <x>200</x>
                    <y>260</y>
                    <width>181</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="text">
                <string>Go back</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
